### PR TITLE
Sscs 9433 postponement req

### PR DESCRIPTION
**Pull request by gmikerlemos**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-9433

### Change description ###

Clerk-Submit Postponement Request (Letter): Action Further Evidence Event

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validsendtointerloc/ValidSendToInterlocAboutToStartHandler.java [hunk 1]
```diff
@@ -35,16 +38,24 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
...
         return new PreSubmitCallbackResponse<>(sscsCaseData);
     }
 
     private void setSelectWhoReviewsCase(SscsCaseData sscsCaseData) {
         List<DynamicListItem> listOptions = new ArrayList<>();
-        listOptions.add(new DynamicListItem("reviewByTcw", "Review by TCW"));
-        listOptions.add(new DynamicListItem("reviewByJudge", "Review by Judge"));
+        listOptions.add(new DynamicListItem(REVIEW_BY_TCW.getId(), REVIEW_BY_TCW.getLabel()));
+        listOptions.add(new DynamicListItem(REVIEW_BY_JUDGE.getId(), REVIEW_BY_JUDGE.getLabel()));
+        listOptions.add(new DynamicListItem(POSTPONEMENT_REQUEST_INTERLOC_SEND_TO_TCW.getId(), POSTPONEMENT_REQUEST_INTERLOC_SEND_TO_TCW.getLabel()));
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validsendtointerloc/ValidSendToInterlocAboutToStartHandler.java:50]: Needs to be feature flagged

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validsendtointerloc/ValidSendToInterlocMidEventHandler.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                && (callback.getEvent() == EventType.VALID_SEND_TO_INTERLOC
+                || callback.getEvent() == EventType.ADMIN_SEND_TO_INTERLOCUTORY_REVIEW_STATE);
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
+        final SscsCaseData sscsCaseData = callback.getCaseDetails().getCaseData();
+        final PreSubmitCallbackResponse<SscsCaseData> response = new PreSubmitCallbackResponse<>(sscsCaseData);
+
+        final String requestDetails = sscsCaseData.getPostponementRequest().getPostponementRequestDetails();
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validsendtointerloc/ValidSendToInterlocMidEventHandler.java:45]: What happens if postponement request is blank here, bearing in mind this is a live event? Will they always get an error?

Comment (gmikerlemos) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validsendtointerloc/ValidSendToInterlocMidEventHandler.java]: Is it not mandatory on exui?

Comment (jackmaloney) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validsendtointerloc/ValidSendToInterlocMidEventHandler.java]: Fixed

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validsendtointerloc/ValidSendToInterlocAboutToStartHandler.java (hunk 1 above)
Comment (jackmaloney) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/validsendtointerloc/ValidSendToInterlocAboutToStartHandler.java]: Fixed

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**